src/GraphHopperMatrix.js

```diff
@@ -1,16 +1,20 @@
+var request = require('superagent');
+
 var GHUtil = require("./GHUtil");
 var ghUtil = new GHUtil();
 
 GraphHopperMatrix = function (args) {
     this.host = "https://graphhopper.com/api/1";
-    vehicle = "car";
-    debug = false;
-    data_type = "json";
+    this.vehicle = "car";
+    this.debug = false;
+    //TODO Is there an alternative? Why is this defined as variable?
```

Comment:
Yes we need it for MapMatching and in theory for routing (gpx export). Will remove it here.